### PR TITLE
feat: speedy css tip animated loader post

### PR DESCRIPTION
**Pull request by jh3y**
Short form post for Speedy CSS animated loader ⚡

### CODE REVIEW

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+subhead: Let's make an animated CSS loader with scoped custom properties and animation-timing-function
+date: 2022-09-22
+hero: image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/vzvc5Ep1blYUQMYYnAKT.jpg
+authors:
+  - jheyy
+tags: 
+  - blog
+  - css
+---
+Hop over to [CodePen](https://pen.new) and create a new pen.
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:12]:
```suggestion
---

Hop over to [CodePen](https://pen.new) and create a new pen.
```

Comment (malchata) [reply src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:12]: Total nit, but a line between the front matter and the post is just a bit cleaner to read.

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  <span style="--index: 5"></span>
+  <span style="--index: 6"></span>
+  <span style="--index: 7"></span>
+  <span style="--index: 8"></span>
+  <span style="--index: 9"></span>
+</div>
+
+```
+
+Bonus points for using a generator(Pug 👇) to configure the number of lines.
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:32]:
```suggestion
You can also use a generator ([Pug](https://pugjs.org/api/getting-started.html)) to configure the number of lines:
```

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+```pug
+- const COUNT = 10
+.loader(style=`--count: ${COUNT}`)
+  - let i = 0
+  while i < COUNT
+    span(style=`--index: ${i}`)
+    - i++
+```
+
+Give our loader some styles.
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:43]:
```suggestion
Give our loader some styles:
```

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+hero: image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/vzvc5Ep1blYUQMYYnAKT.jpg
+authors:
+  - jheyy
+tags: 
+  - blog
+  - css
+---
+Hop over to [CodePen](https://pen.new) and create a new pen.
+
+Create the markup for our loader. Note the use of inline custom properties!
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:14]:
```suggestion
Create the markup for our loader. Note the use of inline custom properties:
```

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  while i < COUNT
+    span(style=`--index: ${i}`)
+    - i++
+```
+
+Give our loader some styles.
+
+```css
+loader {
+  --size: 10vmin;
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:47]:
```suggestion
  --size: 10vmin;

```

Comment (malchata) [reply src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:47]: I've seen this enforced by stylelint in the past. I think it helps with readability to put variable declarations at the top of a CSS rule and then separate the following rules with a line.

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  height: var(--size);
+  position: relative;
+  width: var(--size);
+}
+```
+
+Position our lines using absolute positioning and a combo of `calc` with `transform`.
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:54]:
```suggestion
Position our lines using absolute positioning and a combination of `calc` with `transform`:
```

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%)
+             rotate(calc(((360 / var(--count)) * var(--index)) * 1deg))
+             translate(0, -125%);
+  width: 10%;
+}
+```
+
+Apply an opacity based on the `--index`.
```
Comment (malchata) [inline src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:70]:
```suggestion
Apply an opacity based on the `--index`:
```

**Review by malchata — APPROVED** [review]
LGTM, modulo a couple nits.

File: src/site/content/en/blog/speedy-css-tip-animated-loader/index.md (hunk 1 above)
Comment (jh3y) [reply src/site/content/en/blog/speedy-css-tip-animated-loader/index.md:12]: All good. I removed it thinking it had added extra whitespace but don't think that was the case 😅

**Comment by jh3y** [comment]
Thanks for giving this a quick review @malchata 🙏  All great points! Thanks.

**Comment by malchata** [comment]
No sweat. Let me know when you need to merge, or if you can merge, merge when ready!

**Comment by jh3y** [comment]
I do not have those powers ⚡️😅🙏

**Comment by malchata** [comment]
I'll merge once the workflow jobs are finished running.

**Comment by jh3y** [comment]
I've got the power! ⚡ 

power

**Pull request merged**